### DISCUSSION TITLE
Error code

### DISCUSSION THREAD

**Discussion by POSHDEGREE** [aclap-dev/video-downloadhelper#516] [Help and Q&A]
### Your question

I purchased the full version in order to download some audio books from voxa.ro, however I get error. here is one of them:
{
  "version": "9.0.1.8",
  "target": "google",
  "channel": "beta",
  "page_url": "https://voxa.ro/audiobooks/anna-karenina-30?player=true",
  "error": {
    "details": "SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 000001a35b5cb380] trun track id unknown, no tfhd was found\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 000001a35b5cb380] error reading header\r\n[in#0 @ 000001a35b5cb1c0] Error opening input: Invalid data found when processing input\r\nError opening input file https://voxamediaservices-euwe.streaming.media.azure.net/e3d0f3d5-a614-49bb-9d74-a142fe143d04/484521.ism/QualityLevels(192000)/Fragments(aac_und_2_192000_1_1=962560000,format=mpd-time-csf,encryption=cenc).\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_4359506493856466",
    "error": "coapp_failure",
    "id": "coapp_failure_fe76cda5-f710-43e7-8e63-3f9274bbb4db"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.8",
  "target": "google",
  "channel": "beta",
  "page_url": "https://voxa.ro/audiobooks/anna-karenina-30?player=true",
  "error": {
    "details": "SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 000001a35b5cb380] trun track id unknown, no tfhd was found\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 000001a35b5cb380] error reading header\r\n[in#0 @ 000001a35b5cb1c0] Error opening input: Invalid data found when processing input\r\nError opening input file https://voxamediaservices-euwe.streaming.media.azure.net/e3d0f3d5-a614-49bb-9d74-a142fe143d04/484521.ism/QualityLevels(192000)/Fragments(aac_und_2_192000_1_1=962560000,format=mpd-time-csf,encryption=cenc).\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_4359506493856466",
    "error": "coapp_failure",
    "id": "coapp_failure_fe76cda5-f710-43e7-8e63-3f9274bbb4db"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user, like you.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.13a1 beta, CoApp 2.0.19.

I visited your Anna Karenina audio book & VDH was perfectly happy to download what it found.  What it found was a 5-minute fragment of the apparently over 36 hours of the book.  I'm not sure what language the book is being read in.  I ran some of the text on the page through Google Translate & it told me it's Romanian, so I'm going to assume that's the language of the file I got.  I am going to guess that if you register for the site, you can access the full 36 hours of the audio novel.  I'm not going to register.  Aside from the fact that I don't register for random web sites just to investigate problems for fellow users, I don't speak Romanian.

There is one large difference between our setups.  You're using Chrome.  I'm using Firefox.  You should try Firefox on this site.  You say you bought a VDH license.  This is a commendable thing.  I'm sure the VDH folks appreciate your contribution.  But you should understand that the only thing a license does for you is it skips the step when VDH writes a watermark into a video.  Video.  This isn't a video.  There is no video in which to write a watermark.  You don't need a license to download this content, not with Chrome, not with Firefox.  So if you happen to have Firefox, install the latest VDH beta for Firefox & see whether that works.  This would not be the first time Firefox handled a web site better than Chrome.
